modules/javafx.controls/src/test/java/test/javafx/scene/control/TableCellTest.java

```diff
@@ -322,6 +323,26 @@ private void test_rt36715_cellPropertiesMirrorTableColumnProperties(
         cell.setSkin(new TableCellSkin(cell));
     }
 
+    @Test
+    public void testCellInUneditableRowIsNotEditable() {
```

Comment:
I recommend adding tests for cell.startEdit() using all combinations of TableView, TableColumn and TableRow editable states.

Reply:
done